lib/trino-orc/src/test/java/io/trino/orc/OrcTester.java

```diff
@@ -985,7 +986,7 @@ public static void writeOrcColumnHive(RecordWriter recordWriter, Type type, Iter
                 (StandardStructObjectInspector) objectInspector,
                 ImmutableList.of(type),
                 StreamSupport.stream(
-                        Spliterators.spliteratorUnknownSize(values, Spliterator.ORDERED), false)
+                                Spliterators.spliteratorUnknownSize(values, Spliterator.ORDERED), false)
```

Comment:
The extra spacing is probably related to the previous commit.

Reply:
yeah, something went wrong with formatting. fixed.